### PR TITLE
Docker mounts for v1.7

### PR DESCRIPTION
**Pull request by ricarkol**
fix for #90 

It includes an `autopep8 -a` on dockercontainer.py and a new unit tests in test_dockercontainer.py.

### CODE REVIEW

File: crawler/dockercontainer.py [hunk 1]
```diff
@@ -270,9 +285,9 @@ def _parse_log_locations(self, var=None):
         data = None
         try:
             data = misc.GetProcessEnv(container.pid)[var]
-	    log_locations = data.split(',')
-	    logs = [{'name': name.strip(), 'type': None}
-		    for name in log_locations]
+            log_locations = data.split(',')
+            logs = [{'name': name.strip(), 'type': None}
```
Comment (sastryduri) [inline crawler/dockercontainer.py]: This is more pythonic, also pylint measures number of local variables as a quality measure (flags warnings if number exceeds say 15).

 logs = [{'name': name.strip(), 'type': None}
                    for name in  misc.GetProcessEnv(container.pid)[var].split(',')]

File: crawler/dockercontainer.py [hunk 2]
```diff
@@ -364,17 +378,17 @@ def _set_logfiles_links_source_and_dest(self,
...
-		self.long_id, self.inspect)
-	    name = 'docker.log'
-	    docker_log_dest = os.path.join(host_log_dir, name)
-	    logs_list.append({'name': name,
-			      'type': None,
-			      'source': docker_log_source,
-			      'dest': docker_log_dest})
+            docker_log_source = get_docker_container_json_logs_path(
+                self.long_id, self.inspect)
+            name = 'docker.log'
```
Comment (sastryduri) [inline crawler/dockercontainer.py]: name = 'docker.log' is redundant
make docker.log a constant if appropriate

**Comment by ricarkol** [comment]
Thanks for the review @sastryduri addressed the comments. Squashed the new commit.

File: crawler/dockercontainer.py [hunk 3]
```diff
@@ -269,10 +286,8 @@ def _parse_log_locations(self, var=None):
         logs = []  # list of maps {name:name,type:type}
         data = None
```
Comment (sastryduri) [inline crawler/dockercontainer.py]: data = None is not required, it is no longer used

**Pull request merged**